pkg/apis/jvmbuildservice/v1alpha1/systemconfig_types.go

```diff
@@ -47,8 +47,8 @@ type SystemConfigList struct {
 
 const (
 	KonfluxGitDefinition          = "https://raw.githubusercontent.com/konflux-ci/build-definitions/refs/heads/main/task/git-clone/0.1/git-clone.yaml"
-	KonfluxPreBuildDefinitions    = "https://raw.githubusercontent.com/redhat-appstudio/jvm-build-service/main/deploy/tasks/pre-build.yaml"
+	KonfluxPreBuildDefinitions    = "https://raw.githubusercontent.com/rnc/jvm-build-service/TOOLING/deploy/tasks/pre-build.yaml"
```

Comment:
Can you use some shared repository instead of personal (rnc)?

Reply:
https://github.com/redhat-appstudio/jvm-build-service/pull/2376